geomesa-tools/bin/geomesa

```diff
@@ -121,41 +48,21 @@ elif [[ (-z "$ACCUMULO_CONF_DIR") || (-z "$HADOOP_CONF_DIR")]]; then
     echo >&2 ""
 fi
 
-function findJars() {
-    home="$1"
-    CP=""
-    if [ -n "$home" -a -d "$home" ]; then
-        for jar in $(find ${home} -name "*.jar"); do
-            if [[ ("$2" != "true") || (("$jar" != *"test"*) && ("$jar" != *"slf4j"*)) ]]; then
-                if [ "$jar" = "${jar%-sources.jar}" -a "$jar" = "${jar%-test.jar}" ]; then
-                  CP="$CP:$jar"
-                fi
-            fi
-        done
-        if [ -d "$home/native" ]; then
-            if [ -z "$JAVA_LIBRARY_PATH" ]; then
-                JAVA_LIBRARY_PATH="$home/native"
-            else
-                JAVA_LIBRARY_PATH="$home/native:$JAVA_LIBRARY_PATH"
-            fi
-        fi
-    fi
-    if [ "${CP:0:1}" = ":" ]; then
-        CP="${CP:1}"
-    fi
-    echo $CP
-}
-
 function findSftDirs() {
     DIRS=$(find $GEOMESA_CONF_DIR/sfts -mindepth 1 -type d -exec printf {}: \;)
     echo ${DIRS%:}
 }
+
 # Start constructing GEOMESA_CP (classpath)
 # include geomesa first so that the correct log4j.properties is picked up
 GEOMESA_CP="${GEOMESA_CONF_DIR}:$(findJars $GEOMESA_LIB/common)"
 GEOMESA_CP="${GEOMESA_CP}:${ACCUMULO_CONF_DIR}:${HADOOP_CONF_DIR}"
 GEOMESA_CP="${GEOMESA_CP}:${GEOMESA_CONF_DIR}:$(findSftDirs)"
-ACCUMULO_CP="$(findJars $ACCUMULO_LIB true)"
+# Add user defined directories to the classpath
+if [[ -z "$CUSTOM_CONF_DIR"  ]]; then
```

Comment:
maybe rename this from ```CUSTOM_CONF_DIR``` to something like GEOMESA_EXTRA_CLASSPATH or something since ```CUSTOM_CONF_DIR``` I read as referring to a geomesa conf dir